svgscreen/template.svg

```diff
@@ -3,13 +3,11 @@ Tries to generate SVG code comptible with browers and inkscape.
 For some reason inkscape seems to not like CSS for rect fill etc that much
 so currently using <use> instead.
 */ -}}
-{{- $width := mul $.TerminalWidth .CharacterBoxSize.Width -}}
-{{- $height := mul $.NrLines .CharacterBoxSize.Height -}}
 {{- /*
 xml:space="preserve" is used to forces some svg readers (ex inkscape) to not collapse
 whitespace only tspan:s preventing underline to be properly rendered.
 */ -}}
-<svg viewBox="0 0 {{$width}} {{$height}}" width="{{$width}}" height="{{$height}}" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve">
+<svg width="{{$.TerminalWidth}}ch" height="{{$.NrLines}}em" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve">
```

Comment:
I wonder how it would look like to use `encoding/xml` instead of a xml template? 🤔  maybe can experiment with that later. One nice thing about using template is that you get a nice overview i think

Reply:
I like that it's using a template, I just found there was a bit much logic in it. My favorite approach would be to keep the logic in the program code and make the template just set up the SVG elements according to the svgscreen struct.